project_issue_task_sync/README.rst

```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Sync Issues and Tasks
+=====================
+
+This module syncs assigned user and state from task to issue and viceversa.
+If a task is created and does not have a issue a related issue will be
+generated. Projects may or may not have this syncing function, the activation
+of sync between issues and task may be activated on the project form by
+selecting the checkbox "sync issues and tasks". By default projects that use
+tasks and issues will be synced.
```

Comment:
(Functional Q) I'm puzzled why we need to always have Issue/Task pairs.
For example, I have used a workflow like this:
- Create Issue
- Triage Issue: close it or create a Task
- Do the Task, and keep it in sync with the origin issue

In this case you can have Issues without Tasks.
So I think that requiring the Issue - Task pair to always be true is too specific of a particular use case.
What do you think?